testing/tests/DevExpress.ui.widgets/treeViewParts/events.js

```diff
@@ -1010,3 +1010,20 @@ QUnit.test('ContentReady event rise once when the data source is remote by first
 
     assert.strictEqual(contentReadyHandler.callCount, 1, 'onContentReady was first time');
 });
+
+QUnit.test('Should not throw error if event does not have the \'originalEvent\' property', function(assert) {
+    if(QUnit.urlParams['nojquery']) {
+        assert.ok(true);
```

Comment:
This testcase works in `nojquery` mode as well for me. Why should we disable it?

Reply:
I've checked it. It does not work. I added a note to the test